onnx_chainer/functions/converter.py

```diff
@@ -0,0 +1,54 @@
+class FunctionConverterParams(object):
+
+    def __init__(
+            self, func=None, opset_version=None, input_names=None,
+            output_names=None, context=None, parameters=None):
```

Comment:
s/output_names/num_outputs/? Here, and elsewhere.

Reply:
It's point I want to discussion about. Customized converter does not have to use`onnx_chainer.onnx_helper.make_node` instead of `onnx.helper.make_node` and latter function requires `output_names`, so I selected more general one. However output name renamed after generating nodes anyway, so looks nonsense care currently.

Reply:
Ah, sorry. I read the code wrongly. I thought line 53 of this file is the caller of this constructor, but I should have read line 93 of export.py. Yes, I totally agree with your decision. I think it's better to pass `output_names`.